app/models/service_request.rb

```diff
@@ -566,6 +560,16 @@ def is_editable?
     return true
   end
 
+  def has_locked_ssr?
+    sub_service_requests.each do |ssr|
+      if !ssr.can_be_edited?
+        return true
```

Comment:
Could you call 'map' and then 'any?'

> On Jun 1, 2016, at 11:21 AM, kyle-glick <notifications@github.com> wrote:
> 
> In app/models/service_request.rb:
> 
> > @@ -566,6 +560,16 @@ def is_editable?
> >      return true
> >    end
> >  
> > +  def has_locked_ssr?
> > +    sub_service_requests.each do |ssr|
> > +      if !ssr.can_be_edited?
> > +        return true
> I tried removing the returns, and the functionality did not work the same way. The return appears to be needed
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>